targets/jm-temp-humidity-18-1.0/config.mk

```diff
@@ -0,0 +1,2 @@
+MCU = STM32F030x4
```

Comment:
wrong mcu?

Reply:
Sorry, didn't commit everything. Should be correct now